### PR TITLE
support for saving and loading of hdd image

### PR DESCRIPTION
**Pull request by nadvornik**

### CODE REVIEW

File: tests/shutdown/shutdown.pm [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+use strict;
+use base 'basetest';
+use testapi;
+
+sub run {
+    my $self = shift;
+
+    if (get_var("SAVE_IMAGE_HDD_1")) {
+        upload_image(1, get_var("SAVE_IMAGE_HDD_1"));
```
Comment (coolo) [inline tests/shutdown/shutdown.pm]: as Ondrej's test uses STORE_ASSET, shouldn't we stick to the STORE term? 

Now STORE_ASSET is very generic and SAVE_IMAGE_HDD_1 very specific. Perhaps you can come up with something in the middle?

@sysrich - any suggestions? One is public, one is private - but that's about it IMO

Comment (sysrich) [inline tests/shutdown/shutdown.pm]: STORE_IMAGE_$num ?

Comment (nadvornik) [inline tests/shutdown/shutdown.pm]: we have HDD_1 for loading
so for saving we should IMO use HDD_1 suffix too.

What about:
STORE_HDD_1
PUBLISH_HDD_1

**Comment by coolo** [comment]
yeah, like that.

**Comment by coolo** [comment]
please also come up with a good set of templates - e.g. move the sysauth test to boot an image created by minimalx installation.

**Pull request merged**